src/Semantics.js

```diff
@@ -96,6 +96,30 @@ Object.defineProperties(Wrapper.prototype, {
   // Returns the number of children of this CST node.
   numChildren: {get: function() { return this._node.numChildren(); }},
 
+  // Returns the minimum number of children for this CST node, if it's an iterator node.
+  // Otherwise, throws an exception.
+  minNumChildren: {
+    get: function() {
+      if (this.isIteration()) {
+        return this._node.minMatches;
+      }
+      throw new TypeError(
```

Comment:
Mhmm...

The whole discussion started with me being surprised that `ClassDecl = class className (extends className)? (with Formals)? ";"` leads to a semantic action that has 7 instead of 5 arguments (which I see when I look at it). Wanting to know whether something is optional (0-1x) or a repetition (0+ or 1+), I was looking at the grammar's rule body for `ClassDecl` where I only found 5 parts. Matching those 5 to 7 is hard(er) later on so Alex and I were thinking that putting `min` and `max` into the nodes/wrappers. We decided against having the iterator reporting the `operator` token (`?`, `*` or `+`) because future versions of Ohm could have `{2,5}` just like RegExps do.

Thinking about it now, it might make more sense to have `isOptional` instead of `minNumChildren` which will also make sense for any other kind of node (NO, not optional). Yet it provides less information.

Thoughts? @pdubroy @alexwarth (I actually favor `isOptional` now.)